test/parallel/test-async-wrap-getasyncid.js

```diff
@@ -212,9 +213,10 @@ if (common.hasCrypto) {
   const TCP = process.binding('tcp_wrap').TCP;
   const tcp = new TCP();
 
-  const ca = fs.readFileSync(common.fixturesDir + '/test_ca.pem', 'ascii');
-  const cert = fs.readFileSync(common.fixturesDir + '/test_cert.pem', 'ascii');
-  const key = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
+  const ca = fs.readFileSync(fixtures.path('test_ca.pem'), 'ascii');
+  const cert = fs.readFileSync(fixtures.path('test_cert.pem'), 'ascii');
+  const key = fs.readFileSync(fixtures.path('test_key.pem'), 'ascii');
```

Comment:
@jasnell re/ https://github.com/nodejs/node/pull/14332#discussion_r129088694

I don't understand, the test passes for me with this change:

```diff
-  const ca = fs.readFileSync(common.fixturesDir + '/test_ca.pem', 'ascii');
-  const cert = fs.readFileSync(common.fixturesDir + '/test_cert.pem', 'ascii');
-  const key = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
+  const ca = fixtures.readSync('test_ca.pem', 'ascii');
+  const cert = fixtures.readSync('test_cert.pem', 'ascii');
+  const key = fixtures.readSync('test_key.pem', 'ascii');
```